SUMMARY.md

```diff
@@ -0,0 +1,25 @@
+## Table of Contents
+
+* [Connecting to Database](content/ContectingToDatabase/ConnectingToDatabase.q.md) ** - minutes**
+    > Learn how you can run PostgreSQL database operations in PHP using a specific class or command to establish a connection.
+
+* [PHP Data Objects](content/PDOClasses/PDOClasses.q.md) ** - minutes**
+    > In this lesson, you will learn about PHP Data Objects (`PDO`) classes and how to run queries using `PDO::query()`.
+
+* [Prepared Statement](content/PreparedStatement/PreparedStatement.q.md) ** - minutes**
+    > Prepared statement in PDO uses the method `PDO::prepare()` to prepare a statement for execution and return a statement object. Learn how you can run `PDO::prepare()` with its parameters.
```

Comment:
Which word is better to use in this statement __Learn how you can run `PDO::prepare()` with its parameters.__? Is it `run`, `execute`, or `use`? Please check. Thanks.